docs/release-notes/8.0/8.0.35-31.0.md

```diff
@@ -0,0 +1,33 @@
+# Percona XtraBackup 8.0.35-31 (2024-06-26)
+
+Get started with [Quickstart Guide for Percona XtraBackup](..//..//quickstart-overview.md).
+
+## Release highlights
+
+This release adds Percona XtraBackup ARM64 packages and provides bug fixes.
+
+## Bug fix
+
+* [PXB-3283]: When XtraBackup performs a backup and exports a tablespace for tables that have changed their charset-collation before the backup, it creates an incorrect table definition. This incorrect table definition means the table's structure information may not match the data.
```

Comment:
If backup contains tables with temporal datatypes (https://dev.mysql.com/doc/refman/8.0/en/date-and-time-types.html) and their charset/collation changed (ALTER TABLE CONVERT CHARACTER SET) before the backup is taken, xtrabackup export couldn't generate correct configuration (.cfg)files required by the import.